encrypted-media-respec.html

```diff
@@ -2195,7 +2207,7 @@
         </p>
         <p>The CDM MUST ensure that data for a given session is only present in one active unclosed session in any <a def-id="document-concept"></a>.
           In other words, <a def-id="load"></a> MUST fail when there is already a <a>MediaKeySession</a> representing the session specified by the <var>sessionId</var> parameter, either because the object that created it via <a def-id="generateRequest"></a> is still active or it has been loaded into another object via <a def-id="load"></a>.
-          A session MAY only be loaded again after the <a def-id="session-close-algorithm"></a> algorithm has not been run on the object representing it.
+          A session MAY only be loaded again after the <a def-id="session-closed-algorithm"></a> algorithm has not been run on the object representing it.
```

Comment:
Line 2208:
We use "unclosed session" three times in the spec, but this term is not defined. Should we a) replace these with `the <a def-id="session-closed-algorithm"></a> algorithm has been run on this object` or b) add a definition for closed session, possibly using this text, and update all such text as well as these three cases to reference the definition?

(b) might be better from a readability perspective and avoids having to ensure we get the somewhat complex text correct everywhere.